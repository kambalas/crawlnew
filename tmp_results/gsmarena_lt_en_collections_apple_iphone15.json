[
  {
    "name": "Apple iPhone 16 Pro 1TB",
    "price": "1.699,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 1TB",
    "price": "1.597,99",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 1TB (Pažeista pakuotė)",
    "price": "1.540,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 1TB (Ekspozicinė prekė)",
    "price": "1.490,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 512GB",
    "price": "1.418,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 512GB",
    "price": "1.409,85",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 512GB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 512GB (Pažeista pakuotė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 512GB (Pažeista pakuotė)",
    "price": "1.316,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB",
    "price": "1.248,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 512GB (Pažeista pakuotė)",
    "price": "1.215,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB (Pažeista pakuotė)",
    "price": "1.210,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB",
    "price": "1.206,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB (Pažeista pakuotė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB",
    "price": "1.167,50",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "1.155,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB (Pažeista pakuotė)",
    "price": "1.099,00",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 512GB (Damaged packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 512GB",
    "price": "1.052,00",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB",
    "price": "1.044,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB (Pažeista pakuotė)",
    "price": "1.026,00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB (Damaged packaging)",
    "price": "999,00",
    "error": false
  }
]