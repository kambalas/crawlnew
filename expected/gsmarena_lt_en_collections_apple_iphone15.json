[
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Apple iPhone 14 Plus 256GB (Damaged packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 256GB",
    "price": "719",
    "error": false
  },
  {
    "name": "Apple iPhone 14 256GB (Damaged Packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro Max 1TB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 128GB Black",
    "price": "688.79",
    "error": false
  },
  {
    "name": "Apple iPhone 15 128GB (Ekspozicinė prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 128GB (Damaged packaging)",
    "price": "650",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro 256GB (Naudota prekė)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16e 128GB",
    "price": "647.96",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro 128GB (Exhibition item)",
    "price": "643",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB (Used product)",
    "price": "630",
    "error": false
  },
  {
    "name": "Apple iPhone 14 128GB",
    "price": "587.81",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro Max 128GB (Used product)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 Pro Max 512GB (Used item)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 128GB (Damaged packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 256GB Midnight",
    "price": "539",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro Max 256GB (Used product)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 Mini 256GB (Damaged packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 128GB (Exhibition item)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 128GB (Damaged Packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone SE 2022 128GB (Damaged packaging)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone SE 2022 64GB",
    "price": "390",
    "error": false
  }
]